stellar_account_prometheus_exporter/exporter.py

```diff
@@ -87,18 +100,24 @@ def do_GET(self):
                     self.error(504, 'Error retrieving data from {}'.format(url))
                     return
                 if not r.ok:
-                    self.error(504, 'Error retrieving data from {}'.format(url))
-                    return
+                    m_account_exists.labels(*account_labels).set(0)
+                    logger.error("Error retrieving data from {}".format(url))
+                    continue
                 if "balances" not in r.json():
-                    self.error(500, "Error - no balances found for account {}".format(account["account_id"]))
-                    return
+                    m_balance_exists.labels(*account_labels).set(0)
+                    logger.error("Error - no balances found for account {}".format(account["account_id"]))
+                    continue
                 if "subentry_count" not in r.json():
-                    self.error(500, "Error - no subentry_count found for account {}".format(account["account_id"]))
-                    return
+                    m_available_balance_exists.labels(*account_labels).set(0)
+                    logger.error("Error - no subentry_count found for account {}".format(account["account_id"]))
+                    continue
```

Comment:
There may be a logic error here. If statement:
`if not r.ok:`
then we'll set one metrics and then `continue`. In this case `m_balance_exists` and `m_available_balance_exists` won't be set at all.

I think it would be the simplest to just have one metric to indicate whether we got the data successfully or not for a given account. If there is _any_ problem we can set it to 0 and let operators figure it out. Data that's partially correct is effectively not correct and having one metric to indicate that is enough I think.

Reply:
Initially my understanding was we wanted to provide a clear picture using granular information. Hence, I had added two additional metrics to achieve that. Idea of having single metrics sounds good too.